bom/pom.xml

```diff
@@ -544,6 +560,40 @@
                   <compilerArgument>-AtranslationFilesPath=${project.basedir}/target/generated-translation-files</compilerArgument>
                </configuration>
             </plugin>
+            <plugin>
+               <groupId>org.apache.karaf.tooling</groupId>
+               <artifactId>karaf-maven-plugin</artifactId>
+               <version>${version.karaf-maven-plugin}</version>
```

Comment:
I think this should be in the parent POM, that's where we usually put the plugins.